src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -294,6 +314,10 @@ var ReactElementValidator = {
 
     validatePropTypes(element);
 
+    if (__DEV__) {
```

Comment:
this was reference to ReactDebugCurrentFrame removed in c508d90 not sure why diff isn't showing up, just fyi
```diff
-  ReactDebugCurrentFrame.element = null;
```